components/admin_console/billing/company_info_display.tsx

```diff
@@ -67,8 +67,8 @@ const CompanyInfoDisplay: React.FC = () => {
     }
 
     let body = noCompanyInfoSection;
-    const address = companyInfo.company_address.line1 ? companyInfo.company_address : companyInfo.billing_address;
-    if (address.line1) {
+    const address = (companyInfo.company_address && companyInfo.company_address.line1) ? companyInfo.company_address : companyInfo.billing_address;
+    if (address && address.line1) {
```

Comment:
Could this just be:

```suggestion
    const address = companyInfo.company_address?.line1 ? companyInfo.company_address : companyInfo.billing_address;
    if (address?.line1) {
```
?